mkdocs.yml

```diff
@@ -70,6 +70,7 @@ nav:
           - Concepts: 'features/search/concepts.md'
           - Search Architecture: 'features/search/architecture.md'
           - Search Engines: 'features/search/search-engines.md'
+          - Collators: 'features/search/collators.md'
```

Comment:
Hey @awanlin awesome to see some docs updates. More then happy to review these and update the docs structure, just wanted to make sure you have seen the concept docs we have which is meant to be there to explain different concepts where we have a section for collators https://backstage.io/docs/features/search/concepts#collators. I wonder if we now have some overlap between the concept docs and how to guides with these changes you have made. It also looks like the new collators docs include how to guides so I'm interested in knowing why you wanted to separate them out. Thank you!

Reply:
It really came down to needing a place to make this content more visible as it was coming up on Discord regularly and following what was done with Search Engines. The Concepts page seemed like the wrong place to add installation and configuration information and the How To Guides is very long and not as easy to find what you are looking for as it could be.

Reply:
Makes sense to me, lets try it out! Thanks a lot!